src/SWP/Component/Bridge/Model/RenditionInterface.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Superdesk Web Publisher Bridge Component.
+ *
+ * Copyright 2017 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2017 Sourcefabric z.ú
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\Component\Bridge\Model;
+
+use SWP\Component\Common\Model\SoftDeletableInterface;
+use SWP\Component\Storage\Model\PersistableInterface;
+
+interface RenditionInterface extends PersistableInterface, SoftDeletableInterface
+{
+    /**
+     * @return mixed
+     */
+    public function getId();
+
+    /**
+     * @param mixed $id
+     */
+    public function setId($id);
+
+    /**
+     * @return string
+     */
+    public function getName(): string;
```

Comment:
change minimum php version in component to 7.0 if you use return types (i'm ok with it).